### ISSUE TITLE
Update ITensorTDVP compat to 0.0.4

### ISSUE THREAD

**Issue by emstoudenmire** [mtfishman/ITensorQTT.jl#10]
I'm not able to install ITensorQTT currently because I get the following message:
```
ERROR: Unsatisfiable requirements detected for package ITensorTDVP [25707e16]:
 ITensorTDVP [25707e16] log:
 ├─possible versions are: 0.0.4 or uninstalled
 ├─restricted to versions 0.0.3 by ITensorQTT [574676e4] — no versions left
 │ └─ITensorQTT [574676e4] log:
 │   ├─possible versions are: 0.0.1 or uninstalled
 │   └─ITensorQTT [574676e4] is fixed to version 0.0.1
 └─ITensorTDVP [25707e16] is fixed to version 0.0.4
```

**Comment by mtfishman** [comment]
That's unfortunate. The fact that ITensorTDVP has versions `0.0.*` makes the default version compatibility a bit too strict. From the side of ITensorQTT we should be able to define the compat entry as `ITensorTDVP = "0.0"` as described here: https://pkgdocs.julialang.org/v1/compatibility/#Caret-specifiers

We could make the next version of `ITensorTDVP` `0.1` to make this work better in general, but as we discussed we should probably just move the code from that repository to `ITensorNetworks.jl` anyway.

**Comment by mtfishman** [comment]
Could you check if this is fixed by https://github.com/mtfishman/ITensorQTT.jl/commit/f9ca92354582c6090ed40757b8ec130112eb2463?

**Comment by emstoudenmire** [comment]
Great. That seems to have fixed it. I was able to do `add https://github.com/mtfishman/ITensorQTT.jl` now.

One minor thing I did notice is that, in the process the package manager downgraded KrylovKit from 0.6.0 to 0.5.4. Or maybe it installed both versions; I'm not sure and I haven't seen it do this before. Maybe what happened is that the 0.5.4 is able to satisfy all packages' constraints whereas the 0.6.0 wasn't.
